src/handlers.ts

```diff
@@ -190,7 +190,7 @@ export const commands: Array<Command> = [
             .filter(({ farmer }) => farmer !== "BANK")
             .map(async (entry) => {
               const user = await interaction.client.users.fetch(entry.farmer);
-              return `- ${balances.indexOf(entry.amount) + 1}. ${user.username}: ${entry.amount}`;
+              return `${balances.indexOf(entry.amount) + 1}\. ${user.username}: ${entry.amount}`;
```

Comment:
```suggestion
              return `${balances.indexOf(entry.amount) + 1}\\. ${user.username}: ${entry.amount}`;
```
Because you need a raw backslash here, you need to escape the backslash.

Before vs after:

![image](https://github.com/user-attachments/assets/62af33f4-41a0-4286-827a-9674799d3af4)

Reply:
Ah yeah, I did briefly ask myself if I should double backslash but was not about to set up a test env for it < 3

thanks :)